autoload/codi.vim

```diff
@@ -693,3 +694,9 @@ function! codi#run(bang, ...)
     return s:codi_spawn(filetype)
   endif
 endfunction
+
+function! codi#log(message)
```

Comment:
I agree it should not be manually invoked. I made it this way so it could be called from different VimL scripts in this plugin. But on second thought, I think you are correct. I think it should remain script local until a situation actually arises where a different script needs to log something. That is more in harmony with [evolutionary design](https://vimeo.com/151616935), which I believe in.